src/protections/NsfwProtection.ts

```diff
@@ -45,25 +45,51 @@ export class NsfwProtection extends Protection {
 
     public async handleEvent(mjolnir: Mjolnir, roomId: string, event: any): Promise<any> {
         if (event['type'] === 'm.room.message') {
-            const content = event['content'] || {};
-            const msgtype = content['msgtype'] || 'm.text';
-            const isMedia = msgtype === 'm.image';
+            let content = JSON.stringify(event['content']);
+            content = content.replace(/"|{|}/g, '')
```

Comment:
this could do with a comment for what it's fixing about the input (I'm not sure myself).